src/typeDefs/withdrawMutations.ts

```diff
@@ -0,0 +1,37 @@
+import { gql } from 'apollo-server-express';
+
+export default gql`
+  """
+  Withdraw input arguments
+  """
+  input WithdrawInput {
+    """
+    Withdrawing account identifier
+    """
+    accountId: String!
+
+    """
+    Withdrawing amount (account currency will be used)
+    """
+    amount: NonNegativeInt!
+  }
+
+  """
+  Special type for withdraw response
+  """
+  type WithdrawResponse {
+    """
+    Withdraw transaction identifier
+    """
+    transactionID: ID!
```

Comment:
```suggestion
    transactionId: ID!
```
до этого везде Id, так что наверное и тут надо также

Reply:
в других местах тоже надо поменять